certified-connectors/PDFco/apiDefinition.swagger.json

```diff
@@ -649,52 +880,7791 @@
+                  "title": "Pages"
+                },
+                "inline": {
+                  "type": "boolean",
+                  "description": "",
+                  "title": "Inline",
+                  "default": true
+                },
+                "wordMatchingMode": {
+                  "type": "string",
+                  "description": "Values can be either ‘SmartMatch’, ‘ExactMatch’, or ‘None’.",
+                  "title": "Word Matching Mode",
+                  "x-ms-visibility": "advanced",
+                  "enum": [
+                    "",
+                    "SmartMatch",
+                    "ExactMatch",
+                    "None"
+                  ]
+                },
+                "password": {
+                  "type": "string",
+                  "description": "Password of the PDF file",
+                  "title": "Password",
+                  "x-ms-visibility": "advanced"
+                },
+                "async": {
+                  "type": "boolean",
+                  "description": "Set async to true for long processes to run in the background, API will then return a jobId which you can use with JobCheck module to check the status of the process and retrieve the output while you can proceed with other tasks without waiting for this process to finish.",
+                  "title": "Async",
+                  "default": false,
+                  "x-ms-visibility": "advanced"
+                },
+                "profiles": {
+                  "type": "string",
+                  "description": "Use this parameter to set additional configurations for fine-tuning and extra options. Explore PDF.co knowledgebase (https://developer.pdf.co/api/profiles/index.html) for profile examples.",
+                  "title": "Profiles",
+                  "x-ms-visibility": "advanced"
+                }
+              },
+              "required": [
+                "url",
+                "searchString"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/v1/pdf/documentparser": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "url": {
+                  "type": "string",
+                  "description": "Output URL",
+                  "title": "URL",
+                  "x-ms-visibility": "important"
+                },
+                "pageCount": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "",
+                  "title": "Page Count",
+                  "x-ms-visibility": "advanced"
+                },
+                "error": {
+                  "type": "boolean",
+                  "description": "",
+                  "title": "Error",
+                  "enum": [
```

Comment:
Please change "true" to true and "false" false, please see the below swagger validation error. 

##[error]The type of the enum value is 'String' but it should be 'Boolean' as specified in the schema. ValuePath: /paths/~1v1~1barcode~1generate/post/responses/200/schema/properties/error/enum

Please refer this schema link. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/apiDefinition.swagger.schema.json#L1002

Please fix all responses in the swagger file.

Reply:
@vmanoharas , I have fixed this.